we skip these tests and this takes ages to timeout with networking disabled
--
--- a/src/test/bin/CMakeLists.txt
+++ b/src/test/bin/CMakeLists.txt
@@ -43,8 +43,8 @@
 
   foreach(image ${images})
     if (NOT EXISTS ${CMAKE_CURRENT_BINARY_DIR}/${image})
-      message(STATUS "Downloading ${images_url}/${image}")
-      file(DOWNLOAD ${images_url}/${image} ${CMAKE_CURRENT_BINARY_DIR}/${image})
+      #message(STATUS "Downloading ${images_url}/${image}")
+      #file(DOWNLOAD ${images_url}/${image} ${CMAKE_CURRENT_BINARY_DIR}/${image})
     endif()
   endforeach()
 
